src/create_isurf.cpp

```diff
@@ -808,22 +947,26 @@ void CreateISurf::comm_neigh_corners(int which)
 
   double *rbuf;
   int nrecv = comm->irregular_uniform_neighs(nsend,proclist,(char *) sbuf,
-                                             ncomm*sizeof(double),(char **) &rbuf);
+                ncomm*sizeof(double),(char **) &rbuf);
 
   // realloc val_ghost if necessary
 
   if (grid->nghost > maxghost) {
+    memory->destroy(inghost);
     memory->destroy(cghost);
     memory->destroy(sghost);
     memory->destroy(ighost);
     maxghost = grid->nghost;
+    memory->create(inghost,maxghost,ncorner,ninner,"createisurf:inghost");
     memory->create(cghost,maxghost,ncorner,"createisurf:cghost");
     memory->create(sghost,maxghost,ncorner,"createisurf:sghost");
-    memory->create(ighost,maxghost,ncorner,nadj,"createisurf:ighost");
+    memory->create(ighost,maxghost,ncorner,ninner,"createisurf:ighost");
   }
 
   // unpack received data into val_ghost = ghost cell corner points
 
+  // NOTE: need to check if hashfilled
```

Comment:
@andrewykhong is this comment still relevant?

Reply:
Nope. I removed the comment in a commit just now